File: src/main/java/org/pixeltime/enchantmentsenhance/commands/MenuCommand.java
package org.pixeltime.enchantmentsenhance.commands;

import org.bukkit.entity.Player;
import org.pixeltime.enchantmentsenhance.events.menu.Menu;
import org.pixeltime.enchantmentsenhance.manager.SettingsManager;

public class MenuCommand extends SubCommand {

    @Override
    public void onCommand(Player player, String[] args) {
        Menu.showEnhancingMenu(player);
    }


    @Override
    public String name() {
        return "menu";
    }


    @Override
    public String info() {
        return "\n&6/enhance menu &7- " + SettingsManager.lang.getString(
                "Help.menu");
    }


    @Override
    public String[] aliases() {
        return new String[]{"menu", "yemian", "ym"};
    }


    @Override
    public String getPermission() {
        return "Enchantmentsenhance.enhance";
    }

}

File: modules/EnchantmentsEnhance-Plugin/src/main/kotlin/org/pixeltime/enchantmentsenhance/command/player/MenuCommand.kt
package org.pixeltime.enchantmentsenhance.command.player

import org.bukkit.entity.Player
import org.pixeltime.enchantmentsenhance.command.SubCommand
import org.pixeltime.enchantmentsenhance.gui.menu.MainMenu

class MenuCommand : SubCommand() {


    override val permission: String
        get() = "Enchantmentsenhance.enhance"

    override fun onCommand(player: Player, args: Array<String>) {
        MainMenu(player).open()
    }


    override fun name(): String {
        return "menu"
    }


    override fun usage(): String {
        return "/enhance menu"
    }


    override fun aliases(): Array<String> {
        return arrayOf("menu", "yemian", "ym")
    }

}
